.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+# Inlcuding the tests with ubuntu to get actual test coverage. The OSX builds skip seccomp tests.
+name: test-coverage
+on: [push, pull_request]
+jobs:
+  validate:
+    name: Validate 
+    runs-on: ubuntu-latest
+    # Pull requests from the same repository won't trigger this checks as they were already triggered by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15'
+      - name: Install libseccomp-dev
+        run: |
+          sudo apt update
+          sudo apt install -y libseccomp-dev
+      - name: unit test
+        run: make test-unit
+      - name: Send the coverage output
+        uses: shogo82148/actions-goveralls@v1
```

Comment:
Do you think we could elaborate on coveralls.io vs codecov.io? I'm kinda more in favor of codecov, but this is just my personal taste. :innocent:

Reply:
I don't have a dog in the fight. I changed it to codecov :)